src/gui/cli/PlayerData.cpp

```diff
@@ -86,7 +122,8 @@ namespace Zappy::GUI {
             {6, [](ItemPacket aInventory) {
                  return aInventory._thystame;
              }}};
-        if (myInventoryMap.find(aSlot) == myInventoryMap.end()) {
+        X if (myInventoryMap.find(aSlot) == myInventoryMap.end())
```

Comment:
what is this X character at the beginning of the line ?

Reply:
miss click xD